src/huntsman/pocs/observatory.py

```diff
@@ -267,26 +267,28 @@ def take_flat_fields(self, camera_names=None, alt=None, az=None, safety_func=Non
 
         self.logger.info('Finished flat-fielding.')
 
-    def take_dark_fields(self,
-                         exptimes,
+    def take_dark_images(self,
+                         exptimes=[],
```

Comment:
It is more common to have `exptimes=None` in the function arguments and then have a statement like
```
if exptimes is None:
    exptimes = list()
```
Somewhere near the start of the function.

Reply:
Or an error as I suggest below.